modules/core/src/lib/layer.js

```diff
@@ -51,7 +52,7 @@ const defaultProps = {
   dataTransform: {type: 'function', value: data => data, compare: false},
   fetch: {
     type: 'function',
-    value: url => fetch(url).then(response => response.json()),
+    value: (url, {layer}) => load(url, layer.getLoadOptions()),
```

Comment:
Q: Should it be possible to specify alternate/additional loaders for a layer? Something for the RFC, I suppose.

Reply:
It will be super convenient if we just support a `loaders` field in the options object.